backend/t4cclient/routes/repositories/__init__.py

```diff
@@ -67,6 +71,76 @@ def get_repositories(db: Session = Depends(get_db), token=Depends(JWTBearer())):
     ]
 
 
+@router.get(
+    "/{project}/revisions", tags=["Repositories"], responses=AUTHENTICATION_RESPONSES
+)
+def get_revisions(
+    project: str, db: Session = Depends(get_db), token=Depends(JWTBearer())
+):
+    remote_refs: dict[str, list[str]] = {}
+    remote_refs["branches"] = []
+    remote_refs["tags"] = []
+
+    git_model = get_primary_model_of_repository(db, project)
+    try:
+        url = git_model.path
+        log.info("Fetch revisions of git-model, %s,  with url: %s", git_model.name, url)
+    except AttributeError:
```

Comment:
When did this occur?

Reply:
I assume it does just occurs if there is no git model assigned. I changed it, please check my changes.